examples/export-data/main.ts

```diff
@@ -1,2 +1,183 @@
-console.log("Hello World")
\ No newline at end of file
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import { dbConnect } from './utils/db-connect';
+import { AnalyticsViewer, AnalyticsLogger, AnalyticsManager } from "bog-analytics";
+import { ClickEventModel } from './models/click-event';
+import { VisitEventModel } from './models/visit-event';
+import { InputEventModel } from './models/input-event';
+import { CustomEventModel } from './models/custom-event';
+import { Project } from './utils/types';
+import { EventEnvironment } from './utils/types';
+import { getProjectIDByName, createProject } from './actions/project'
+
+dotenv.config();
+const CLIENT_KEY = process.env.CLIENT_API_SECRET || '';
+const SERVER_KEY = process.env.SERVER_API_SECRET || '';
+const projectName = "Bits of Good Example Project";
+
+const main = async () => {
+    try {
+        await dbConnect();
+        console.log('Connected to MongoDB');
+
+        // Create the project if it doesn't exist
+        let projectId = await getProjectIDByName(projectName);
+        if (!projectId) {
+            console.log(`Project '${projectName}' does not exist. Creating new project`);
+            
+            const newProject: Partial<Project> = {
+                projectName: projectName,
+                clientApiKey: CLIENT_KEY,
+                serverApiKey: SERVER_KEY,
+                privateData: false,
+                deletionPolicy: -1,
+            };
+            
+            const createdProject = await createProject(newProject);
+            projectId = createdProject._id;
+            console.log(`Created new project with ID: ${projectId}`);
+        } else {
+            console.log(`Project ID found: ${projectId}`);
+        }
+
+        const { clickEvents, visitEvents, inputEvents, customEvents } = await fetchEvents();
+        
+        console.log(`Fetched ${clickEvents?.length} click events`)
+        console.log(`Fetched ${visitEvents?.length} visit events`)
+        console.log(`Fetched ${inputEvents?.length} input events`)
+        console.log(`Fetched ${customEvents?.length} custom events`)
+
+        for (const event of clickEvents || []) {
+            await ClickEventModel.create({
+                eventProperties: event.eventProperties,
+                category: event.category,
+                subcategory: event.subcategory,
+                projectId: projectId,
+                timestamp: event.createdAt ? new Date(event.createdAt) : new Date(),
+            });
+        }
+
+        for (const event of visitEvents || []) {
+            console.log(`event ${event}`)
+            await VisitEventModel.create({
+                eventProperties: event.eventProperties,
+                category: event.category,
+                subcategory: event.subcategory,
+                projectId: projectId,
+                timestamp: event.createdAt ? new Date(event.createdAt) : new Date(),
```

Comment:
change timestamp to createdAt (timestamp: true in MongoDB just means that createdAt and updatedAt are created automatically)

also add in an updatedAt variable because those also exist